packages/angular_devkit/build_angular/test/browser/rebuild_spec_large.ts

```diff
@@ -315,13 +315,6 @@ describe('Browser Builder rebuilds', () => {
   });
 
   it('rebuilds after errors in AOT', async () => {
-    // DISABLED_FOR_IVY - These should pass but are currently not supported
-    if (!veEnabled) {
-      pending('Broken in Ivy: https://github.com/angular/angular/issues/32214');
```

Comment:
This is fixed but not released yet: https://github.com/angular/angular/issues/32214